modules/installation-initializing.adoc

```diff
@@ -362,6 +363,36 @@ endif::[]
 ifndef::restricted[]
 . Modify the `install-config.yaml` file. You can find more information about
 the available parameters in the *Installation configuration parameters* section.
++
+[NOTE]
+====
+If you have any intermediate CA certificates on the engine, verify that the certificates appear in the `/ovirt-config.yaml` file and the `install-config.yaml` file. If they do not appear, add them as follows:
+
+. In `~/.ovirt/ovirt-config.yaml`:
```

Comment:
```
. In the `~/.ovirt/ovirt-config.yaml` file:
```

Reply:
Is this necessary because of style guide requirements? Personally, I think it's unnecessary and anti-minimalism.

Reply:
Yeah, under "Files and directories" in IBM SG.

Reply:
It is also not optimal for the translation team. Too much minimalism removes necessary information.